test/700-udp-traffic-flush.bats

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Regression test for flushing stale conntrack udp entries https://github.com/containers/netavark/issues/1045
+#
+
+load helpers
+
+function prepare_udp_listener() {
+    cat <<EOF > "$NETAVARK_TMPDIR/udp_listen.rs"
+use std::net::UdpSocket;
+use std::io::{self, Write};
+use std::time::Duration;
+
+fn main() {
+    let socket = UdpSocket::bind("0.0.0.0:8080").expect("could not bind to address");
+    socket.set_read_timeout(Some(Duration::from_secs(1))).expect("set_read_timeout call failed");
+    let mut buf = [0; 1024];
+    match socket.recv_from(&mut buf) {
+        Ok((amt, _src)) => {
+            io::stdout().write_all(&buf[..amt]).unwrap();
+        }
+        Err(_) => {
+            std::process::exit(1);
+        }
+    }
+}
+EOF
+    rustc "$NETAVARK_TMPDIR/udp_listen.rs" -o "$NETAVARK_TMPDIR/udp_listen"
+}
+
+function start_sticky_udp_spammer() {
+    local target_port=10080
+    local message=$1
+    
+    cat <<EOF > "$NETAVARK_TMPDIR/spammer.sh"
+#!/bin/bash
+# Open UDP connection to target on FD 3. 
+# Keeps Source Port constant.
+exec 3<>/dev/udp/127.0.0.1/$target_port
+while true; do
+    echo -n "$message" >&3
+    sleep 0.1
+done
+EOF
+    chmod +x "$NETAVARK_TMPDIR/spammer.sh"
+    run_in_host_netns "$NETAVARK_TMPDIR/spammer.sh" &
+    SPAMMER_PID=$!
+}
+
+function run_udp_flush_test() {
+    local config_file="${TESTSDIR}/testfiles/bridge-udp-stale-conntrack.json"
+    local msg="payload_$$"
+
+    prepare_udp_listener
+
+    start_sticky_udp_spammer "$msg"
+
+    # avoid race condition and allow conntrack entry creation
+    sleep 0.5
+
+    run_netavark --file "$config_file" setup $(get_container_netns_path)
+
+    run_in_container_netns "$NETAVARK_TMPDIR/udp_listen"
+    
+    local container_output="$output"
+    local container_status="$status"
+
+    kill $SPAMMER_PID
+    wait $SPAMMER_PID || true
+    run_netavark --file "$config_file" teardown $(get_container_netns_path)
+
+    if [ "$container_status" -ne 0 ]; then
+        fail "Container did not receive UDP traffic (listener timeout). The stale conntrack entry likely persisted."
+    fi
+    
+    if [[ "$container_output" != *"$msg"* ]]; then
+        fail "Container received traffic but payload did not match. Expected '$msg', got '$container_output'"
+    fi
```

Comment:
we are using our `assert` helper function to match things which produce more readable tests and errors

`assert "$container_output" =~ "$msg"` "received proper payload"